config/locales/en.yml

```diff
@@ -38,6 +39,23 @@ en:
         Each deliverable should provide meaningful value towards a goal.
         Depending on the type of mission you're on, value could be money, time, happiness or something else.
         If you're planning on going to the moon, deliverables may be something like this.
+    show:
+      now-what_html:
+        Cool! so you're delivering '%{deliverable_name}',
+        what are the requirements to achieve this?
+      add-requirement:
+        Add a requirement.
+      edit: (edit)
+      requirements: Requirements
+      requirement_html: "%{name} %{edit_link}"
+
+  requirements:
+    new:
+      title: Add a Requirement
+      description:
+        what needs to be done?
+        Each reuirement should represent an actionable item.
```

Comment:
Do you think we need tasks as well -- or do you think that'll nest things too deeply?

## Without tasks

* Move apartment
  * Secure new one
    * Appointment 1 @ x
    * Appointment 2 @ x
    * Put down first months' rent
  * Get things to the new one
    * Hire movers
    * Buy boxes
    * Buy bubble-wrap
    * Pack stuff

## With tasks

* Move apartment
  * Secure new one
    * Find new one 
      * Appointment 1 @ x
      * Appointment 2 @ x
    * Pay for it
      * Put down first months' rent
  * Get things to the new one
    * Get supplies
      * Buy boxes
      * Buy bubble-wrap
    * Move things
      * Hire movers
      * Pack